documentation/modules/operators/ref-operator-cluster-feature-gate-releases.adoc

```diff
@@ -24,10 +24,12 @@ Alpha and beta stage features are removed if they do not prove to be useful.
   It is now permanently enabled and cannot be disabled.
 * The `KafkaNodePools` feature gate moved to GA stage in Strimzi 0.41.
   It is now permanently enabled and cannot be disabled.
+  To use the Kafka Node Pool resources, you still need to use the `strimzi.io/node-pools: enabled` annotation on the `Kafka` custom resources.
 * The `UnidirectionalTopicOperator` feature gate moved to GA stage in Strimzi 0.41.
   It is now permanently enabled and cannot be disabled.
 * The `UseKRaft` feature gate moved to GA stage in Strimzi 0.42.
-It is now permanently enabled and cannot be disabled.
+  It is now permanently enabled and cannot be disabled.
+  To use KRaft (ZooKeeper-less Apache Kafka), you still need to use the `strimzi.io/kraft: enabled` annotation on the `Kafka` custom resources or migrate from an existing ZooKeeper-based cluster using the `strimzi.io/kraft: migration` annotation.
```

Comment:
```suggestion
  To use KRaft (ZooKeeper-less Apache Kafka), you still need to use the `strimzi.io/kraft: enabled` annotation on the `Kafka` custom resources or start the migration from an existing ZooKeeper-based cluster by using the `strimzi.io/kraft: migration` annotation and following the procedure.
```

I am making this suggestion because the `migration` value will just start the migration, but the next phase will still be applying the `enabled` value.

Reply:
This is the exact text you wanted to have in the CHANGELOG 😉

Reply:
maybe we should update the CHANGELOG as well then ;-)

Reply:
I updated it and just left the migration reference generic without mentioning any specific annotation.